ggml-sycl.cpp

```diff
@@ -16802,22 +16809,50 @@ catch (sycl::exception const &exc) {
   std::exit(1);
 }
 
+class host_buffer {
```

Comment:
approved after this is removed

Reply:
yes, remove host_buffer. use malloc/free for thread safe and better performance.